packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -145,9 +145,9 @@ export function createOdspNetworkError(
     let facetCodes: string[] | undefined;
     let innerMostErrorCode: string | undefined;
     if (parseResult.success) {
-        // Log the whole response if it looks like the error format we expect
-        props.response = responseText;
         const errorResponse = parseResult.errorResponse;
+        // logging the error response message
+        props.message = errorResponse.error.message;
```

Comment:
This will overwrite existing message, right? Are we sure it's what we always want? Feels like having two properties for that might be useful (even though I'd agree that having it in `message` is more convenient).
If we go with this route, can we somehow validate (by looking at telemetry) that we are not losing any valuable info?

Reply:
is there an existing message? do you mean the `errorMessage`? that should be a separate property. i will check in unit test

Reply:
This function will create error object that inherits from Error object. it will have Error.message property - see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/message.
props will be added to this error object, and immediately or eventually will overwrite message property (that get recorded in telemetry as Data_error)

Reply:
i see! i will change this to responseMessage